hlx_statics/scripts/delayed.js

```diff
@@ -62,38 +36,9 @@ function loadPrism(document) {
 
 loadPrism(document);
```

Comment:
Saw that you moved loadPrism() to the bottom of file. Awesome! I think this was left here. Let's remove it so we complete the move. 🚀